python/mlc_llm/nn/kv_cache.py

```diff
@@ -1117,11 +1104,13 @@ def batch_decode_paged_kv(
                                             other_o[vec] = O_allreduce[j, ty, tx * VEC_SIZE + vec]
                                         st_m[0] = T.max(st_m[0], other_m[0])
                                         st_d[0] = d_prev[0] * T.exp2(m_prev[0] - st_m[0]) + other_d[0] * T.exp2(other_m[0] - st_m[0])
-                                        for vec in T.serial(VEC_SIZE):
-                                            O_local[vec] = O_local[vec] * T.exp2(m_prev[0] - st_m[0]) + other_o[vec] * T.exp2(other_m[0] - st_m[0])
+                                        exp_mprev[0] = T.exp2(m_prev[0] - st_m[0])
```

Comment:
Just curious, does this help perf improvement? Better vectorization?

Reply:
This will help better vectorization and eventually improves perf.